src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -25,7 +26,18 @@ export default class ScopeModel {
          * @type {EventBus}
          * @private
          */
-        this._eventBus = EventBus;
+        this._eventBus = null;
+
+        /**
+         * Length of PTL lines (aka "vector lines") for all aircraft
+         *
+         * @for ScopeModel
+         * @property _ptlLength
+         * @type {number} length in minutes
+         * @default 0
+         * @private
+         */
+        this._ptlLength = 0;
```

Comment:
if this is a number and is expected to be updated before use, this should actually be `-1`.  assuming only positive numbers are valid for this property, assigning a negative number makes it obvious that this property is invalid

Reply:
It should always default to zero, and thus does not need to be set during instantiation.